enderio-base/src/main/java/com/enderio/base/config/base/common/ItemsConfig.java

```diff
@@ -14,6 +15,29 @@
     public final ForgeConfigSpec.ConfigValue<Integer> LEVITATION_STAFF_ENERGY_USE;
     public final ForgeConfigSpec.ConfigValue<Integer> LEVITATION_STAFF_MAX_ENERGY;
 
+    public final ForgeConfigSpec.ConfigValue<Integer> DARK_STEEL_AXE_ENERGY_PER_FELLED_LOG;
```

Comment:
Maybe we should put upgrades into their own Config class (for debate though, having too many classes is a bad idea too).

Reply:
I actually asked about this in chat a while back, think it got lost in the timezone disconnect. I am split as to what is better, happy to go either way. I chose the same class solely because it was easier